### PR TITLE
Chore(docs): Added release notes for 2022.16

### LINKED ISSUE #8440: Chore: release notes 2022.16

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #8440

**Additional issues**:

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,77 @@ id: Release notes
 section: developer-resources
 ---
 
+## 2022.15 release notes (2022-11-17)
```
Comment (nicolethoen) [inline packages/react-docs/RELEASE-NOTES.md:6]: this title needs to be updated

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -3,6 +3,77 @@ id: Release notes
...
+  - Updated breadcrumb drilldown example ([#8385](https://github.com/patternfly/patternfly-react/pull/8385))
+- **Navigation:**
+  - Fixed navigation flyout accessibility ([#8279](https://github.com/patternfly/patternfly-react/pull/8279))
+- **Pagination:**
+  - Added support for insets ([#8412](https://github.com/patternfly/patternfly-react/pull/8412))
+  - Fixed filter not applying in sticky example ([#8133](https://github.com/patternfly/patternfly-react/pull/8133))
+- **Panel:**
+  - Updated component unit tests to meet new RTL standards ([#8346](https://github.com/patternfly/patternfly-react/pull/8346))
+- **Search input:**
+  - Updated advanced search menu to allow spaces in search form field values (#8369) ([#8372](https://github.com/patternfly/patternfly-react/pull/8372))
```
Comment (nicolethoen) [inline packages/react-docs/RELEASE-NOTES.md:41]: A link is missing to one of the PR numbers

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -3,6 +3,77 @@ id: Release notes
...
+- **Truncate:**
+  - Added `removeFindDomNode` ([#8371](https://github.com/patternfly/patternfly-react/pull/8371))
+- **Navigation:**
+  - Fixed navigation flyout accessibility ([#8279](https://github.com/patternfly/patternfly-react/pull/8279))
+- **Wizard:**
+  - Allowed for async function types for navigation & `isCollapsible` wizard step prop for the next version of the wizard ([#8332](https://github.com/patternfly/patternfly-react/pull/8332))
+  - Converted examples to TypeScript ([#8287](https://github.com/patternfly/patternfly-react/pull/8287))
+
+### Access console extension
+- Prevented running initialization code more than once (#8373) ([#8374](https://github.com/patternfly/patternfly-react/pull/8374))
```
Comment (nicolethoen) [inline packages/react-docs/RELEASE-NOTES.md:59]: A link is missing on this PR number

**Review by nicolethoen — CHANGES_REQUESTED** [review]
Still needs screenshots, too.

File: packages/react-docs/RELEASE-NOTES.md (hunk 3 above)
Comment (tlabaj) [reply packages/react-docs/RELEASE-NOTES.md:59]: The link works for me.  I removed the issue number that was in commit message.

File: packages/react-docs/RELEASE-NOTES.md [hunk 4]
```diff
@@ -3,6 +3,77 @@ id: Release notes
...
+  - Added support for insets ([#8412](https://github.com/patternfly/patternfly-react/pull/8412))
+  - Fixed filter not applying in sticky example ([#8133](https://github.com/patternfly/patternfly-react/pull/8133))
+- **Panel:**
+  - Updated component unit tests to meet new RTL standards ([#8346](https://github.com/patternfly/patternfly-react/pull/8346))
+- **Search input:**
+  - Updated advanced search menu to allow spaces in search form field values (#8369) ([#8372](https://github.com/patternfly/patternfly-react/pull/8372))
+- **Sidebar:**
+  - Updated tests to new react testing library standards ([#8342](https://github.com/patternfly/patternfly-react/pull/8342))
+- **Tabs:**
+ - Converted examples to TypeScript ([#8337](https://github.com/patternfly/patternfly-react/pull/8337))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:45]: Missing space is breaking indentation
```suggestion
  - Converted examples to TypeScript ([#8337](https://github.com/patternfly/patternfly-react/pull/8337))
```

File: packages/react-docs/RELEASE-NOTES.md (hunk 2 above)
Comment (evwilkin) [reply packages/react-docs/RELEASE-NOTES.md:41]: Agree with @nicolethoen - the issue number referenced in the title `(#8369)` being right next to the link to the PR `(#8372)` is confusing, can we remove that text?
```suggestion
  - Updated advanced search menu to allow spaces in search form field values ([#8372](https://github.com/patternfly/patternfly-react/pull/8372))
```

File: packages/react-docs/RELEASE-NOTES.md (hunk 3 above)
Comment (evwilkin) [reply packages/react-docs/RELEASE-NOTES.md]: +1 - same note from Search input above about the issue number in the issue title displaying right next to the issue number linking to the PR.  Your note addresses this, but it looks like the text is still there fyi
```suggestion
- Prevented running initialization code more than once ([#8374](https://github.com/patternfly/patternfly-react/pull/8374))
```

**Review by evwilkin — APPROVED** [review]
LGTM

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**